app/containers/MessageBox/index.js

```diff
@@ -295,17 +295,16 @@ class MessageBox extends Component {
 				const { start, end } = this.component._lastNativeSelection;
 				const cursor = Math.max(start, end);
 				const lastNativeText = this.component._lastNativeText || '';
-				// matches if text either starts with '/' or have (@,#,:) then it groups whatever comes next of mention type
 				const regexp = /(#|@|:|^\/)([a-z0-9._-]+)$/im;
-				const result = lastNativeText.substr(0, cursor).match(regexp);
-				if (!result) {
-					const slash = lastNativeText.match(/^\/$/); // matches only '/' in input
-					if (slash) {
-						return this.identifyMentionKeyword('', MENTIONS_TRACKING_TYPE_COMMANDS);
+				const mentionWithSearch = lastNativeText.substr(0, cursor).match(regexp);
+				if (!mentionWithSearch) {
+					const mentionWithOutSearch = lastNativeText.match(/(#|@|:|^\/)$/);
```

Comment:
Can you explain what's `mentionWithSearch` and `mentionWithOutSearch`?

Reply:
mentionWithSearch is basically a variable to check if the characters #, @, : and / are followed by an alphabet or not. If they are followed by an alphabet then we need to separate those alphabets and use it in the search. Whereas the mentionWithOutSearch is to verify if the user just started with @, #, : and / but has entered any alphabet after that, in this case, we just want to make a search with an empty string.